parts/ch11.tex

```diff
@@ -89,23 +91,28 @@ \subsection{Определенность в обработке зрительн
 
 С сетчатки сигналы, посланные этими нейронами, направляются по оптическому нерву к боковому коленчатому телу, расположенному близко к центру мозга. Там мы находим прямое соответствие поверхности сетчатки, в том смысле, что нейроны коленчатого тела отвечают только на некоторые стимулы, падающие на определенные места сетчатки. В этом смысле коленчатое тело не представляет особого интереса --- это всего-навсего «ретрансляционная станция», и сигналы там не подвергаются дальнейшей обработке (хотя надо всё же отдать ему должное --- коленчатое тело, по-видимому, усиливает чувствительность к световым контрастам). Образ на сетчатке закодирован в схеме сигналов, посылаемых нейронами бокового коленчатого тела, несмотря на то, что нейроны там расположены не на плоскости сетчатки, а в трехмерном блоке. Таким образом, хотя два измерения здесь соответствуют трем, информация тем не менее сохраняется: еще один пример изоморфизма. Возможно, у этого изменения количества измерений есть некий глубинный смысл, которого мы еще не понимаем полностью. Так или иначе, в нашем знании о зрении пока еще так много пробелов, что мы должны не расстраиваться, а радоваться, что нам удалось, хотя бы до определенного предела, понять данный этап.
 
 Из бокового коленчатого тела сигналы поступают обратно в зрительную кору. Здесь они обрабатываются по-новому. Клетки зрительной коры подразделяются на три категории: простые, сложные, и сверхсложные. \emph{Простые} клетки весьма похожи на клетки сетчатки или бокового коленчатого тела они реагируют на освещенные и неосвещенные точки, когда те находятся в контрасте с окружением в определенных местах сетчатки. \emph{Сложные} клетки, с другой стороны, получают информацию от более чем сотни других клеток, и «видят» светлые и темные полосы , расположенные на сетчатке под определенными углами (см. рис. 67). \emph{Сверхсложные} клетки замечают углы, полосы и даже «языки», двигающиеся в определенных направлениях (еще раз см. рис. 67). Эти клетки настолько специализированы, что их иногда называют «сверхсложными клетками высшего порядка».
+%TODO: оформить список, почему не создаётся как отдельный абзац? 
+ \emph{Рис. 67. Ответ на некоторые схемы стимулов различных типов нейронов. 
```

Comment:
One newline in TeX is equal to one space. If you want to end a paragraph (and start a new one), use two newlines (or manually insert `\par` macro to end a paragraph).
However, here, you have to use either \begin{itemize}, or some inline-version of it. Read [`enumitem` documentation](https://ctan.org/pkg/enumitem).

Reply:
Ok, I will try that.